frontend/src/parcelTable.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  HTMLTable,
+} from '@blueprintjs/core';
+
+import {
+  getLulcCodes,
+} from './requests';
+
+export default function ParcelTable(props) {
+  const { parcelTable } = props;
+
+  const [lulcNames, setLulcNames] = useState([]);
+
+  useEffect(async () => {
+    const lulcCodes = await getLulcCodes();
+    setLulcNames(Object.values(lulcCodes));
+  }, []);
+
+  if (!parcelTable) {
+    return <h4>Select a parcel to modify</h4>;
+  }
+
+  const lulcHeader = (
+    <tr key="row1">
+      <td key="col1"> </td>
+      {lulcNames.map((type) => <td key={type}>{type}</td>)}
+    </tr>
+  );
+
+  const rows = [];
+  rows.push(lulcHeader);
+  rows.push(
+    <tr key="row2">
```

Comment:
Does this need to support more than one row if we edit more than one parcel at a time?

Reply:
Probably! But so far we don't support selecting/editing more than one parcel at a time. So that's for later 😄